Mathlib/CategoryTheory/Monoidal/Braided.lean

```diff
@@ -123,6 +129,38 @@ theorem braiding_naturality {X X' Y Y' : C} (f : X ⟶ Y) (g : X' ⟶ Y') :
   rw [tensorHom_def' f g, tensorHom_def g f]
   simp_rw [Category.assoc, braiding_naturality_left, braiding_naturality_right_assoc]
 
+theorem yang_baxter (X Y Z : C) :
+    (α_ X Y Z).inv ≫ ((β_ X Y).hom ▷ Z) ≫ (α_ Y X Z).hom
+    ≫ (Y ◁ (β_ X Z).hom) ≫ (α_ Y Z X).inv
```

Comment:
In mathlib's coding style, we are supposed to write infix operators like `≫` or `≪≫` at the end of the line rather than at the beginning.

Reply:
Fixed. It would be nice if the linter could catch this